kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -356,8 +354,7 @@ parseDebugEquationOptions =
                 \The name of a symbol may be a Kore identifier or a K label, \
                 \which will match any equation where the named symbol \
                 \occurs on the left-hand side. \
-                \The name of a rule is given with the K module name \
-                \as a dot-separated prefix: 'MODULE-NAME.rule-name'."
+                \For multiple equations, use the flag before each identifier."
```

Comment:
I think this would be clearer:

```suggestion
                \This option may be specified multiple times to log multiple equations."
```

The text that was removed,

```
The name of a rule is given with the K module name
as a dot-separated prefix: 'MODULE-NAME.rule-name'
```

is important and it shouldn't be removed.